.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: ci
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
What about using v3 plus setup-python action ?

Reply:
Agree. Something like this would probably work? (Not sure about the requirements though.)
```
steps:
- uses: actions/checkout@v3
- uses: actions/setup-python@v3
  with:
    python-version: '3.6'
    cache: 'pip'
- run: pip install -r requirements.txt
```

Reply:
Haven't used `pip install -r requirements.txt` yet but the other suggestions.